charts/simple-app/values.yaml

```diff
@@ -489,6 +489,20 @@ datadog:
   metricsToScrape:
     - '"*"'
 
+  # -- (`map`) If true, then it will enable application logging to datadog.
+  # To enable datadog logging, datadog.service value must be set as it's
+  # being used as source and service name in datadog logging configuration.
+  scrapeLogs:
```

Comment:
I think I'd rather flatten this out.... `.Values.datadog.scrapeLogs`, `.Values.datadog.scrapeLogsSource`, `.Values.datadog.scrapeLogsProcessingRules. Otherwise, I'd prefer everything to be under `.Values.datadog.scrapeLogs` in a single map like `.Values.datadog.scrapeLogs.enabled`, `.Values.datadog.scrapeLogs.processingRules`, `.Values.datadog.scrapeLogs.source`, etc..